inc/class-importer.php

```diff
@@ -311,11 +322,51 @@ protected function update_post_from_markdown_source( $post_id ) {
 		if ( ! is_null( $title ) ) {
 			$post_data['post_title'] = sanitize_text_field( wp_slash( $title ) );
 		}
+		if ( isset( $yaml['published'] ) ) {
+			$post_data['post_status'] = (bool) $yaml['published'] ? 'publish' : 'draft';
+		}
+		if ( isset( $yaml['date'] ) ) {
+			$post_data['post_date'] = date( 'Y-m-d H:i:s', strtotime( $yaml['date'] ) );
+		}
+		if ( isset( $yaml['author'] ) ) {
+			$user = get_user_by( 'slug', wp_slash( $yaml['author'] ) );
+			if ( $user ) {
+				$post_data['post_author'] = $user->ID;
+			}
+		}
+		if ( isset( $yaml['category'] ) && is_object_in_taxonomy( $this->get_post_type(), 'category' ) ) {
+			wp_set_post_terms( $post_id, wp_slash( $yaml['category'] ), 'category' );
+		}
+		if ( isset( $yaml['categories'] ) && is_object_in_taxonomy( $this->get_post_type(), 'category' ) ) {
+			wp_set_post_terms( $post_id, array_map( 'wp_slash', $yaml['categories'] ), 'category' );
```

Comment:
Is there a reason we need both `category` and `categories` (apart from Jekyll compat, which I'm not too concerned about)?